lib/subvalid/validators/length_validator.rb

```diff
@@ -4,11 +4,19 @@ class LengthValidator
       def self.validate(object, validation_result=ValidationResult.new, *args)
         return unless object
         args = args.to_h
+        message = args.delete(:message)
         args.each do |operator, value|
           case operator
+          when :minimum
+            validation_result.add_error(message || "cannot be shorter than #{value} characters") if object.size < value
           when :maximum
-            validation_result.add_error("is too long, maximum is #{value}") if object.size > value
-            # TODO ALL the other operators from http://guides.rubyonrails.org/active_record_validations.html#length
+            validation_result.add_error(message || "cannot be longer than #{value} characters") if object.size > value
+          when :is
+            validation_result.add_error(message || "should have exactly #{value} characters") if object.size != value
+          when :in, :within
+            validation_result.add_error(
+              message || "should contain #{value.first} to #{value.last} characters"
+            ) unless value.include? object.size
```

Comment:
Trailing conditionals can be a little hard to spot over multi-line code. May I suggest switching to the regular unless/end syntax, which will probably still be three lines. 😁

Reply:
Resolved in dc82724